src/main/webapp/go.js

```diff
@@ -426,7 +434,54 @@ function checkUserDestinationGuess() { //eslint-disable-line
     return;
   }
 
-  // If userGuess is not an exact match, perform entity extraction.
+  // Check to see if userGuess can be used to identify the correct place
+  // using the Places library.
+  checkGuessWithPlaceIDs(userGuess);
```

Comment:
This method is doing more than just checking the guess with place ID.  If the place ID doesn't match, it will also use entity extraction.